ccmlib/cmds/cluster_cmds.py

```diff
@@ -123,6 +122,30 @@ def get_parser(self):
                           help="Supports 'org.apache.cassandra.locator.PropertyFileSnitch','org.apache.cassandra.locator.GossipingPropertyFileSnitch' used only in multidc clusters")
         parser.add_option("--id", type="int", dest="id",
                           help="Allows running multiple clusters in paralell (up to 100) each must have a unique id value 0-99, this will set the ipprefix value if one was not set to 127.0.<id>.")
+
+        parser.add_option('--scylla-package-uri', type="string", dest="scylla_package_uri",
+                          help="The path scylla relocatable package", default=None)
+
+        parser.add_option('--scylla-java-tools-package-uri', type="string", dest="scylla_java_tools_package_uri",
+                          help="The path scylla java tools relocatable package", default=None)
+
+        parser.add_option('--scylla-jmx-package-uri', type="string", dest="scylla_jmx_package_uri",
+                          help="The path scylla jmx relocatable package", default=None)
+
+        parser.epilog = """
+        
+        Examples of using relocatable packages:
+        
+        # create cluster from version uploaded to s3 (of the daily/nightly as example)
+        ccm create fruch --scylla --version unstable/master:239
+        
+        # create cluster with own versions of each package
+        ccm create fruch --scylla --version temp \\
+            --scylla-package-uri=../scylla/temp.tar.gz \\ 
+            --scylla-java-tools-package-uri=../scylla-tools-java/temp.tar.gz \\
+            --scylla-jmx-package-uri=../scylla-jmx/temp.tar.gz
+        """
```

Comment:
what happens if the user will specify only scylla-package-uri ? is this supported ?

Reply:
yes it's supported, as long as a valid version passed down in --version, the other tarball will be download from s3.

you think that might be all or nothing here ? to avoid mixing wrong versions ? or ease of use here is more impotent ?